docs_source_files/content/support_packages/chrome_devtools.en.md

```diff
@@ -279,7 +279,44 @@ public void consoleLogTest() {
 # Please raise a PR to add code sample
   {{< / code-panel >}}
   {{< code-panel language="csharp" >}}
-# Please raise a PR to add code sample
+  
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.DevTools;
+using System;
+using DevToolsSessionDomains = OpenQA.Selenium.DevTools.V91.DevToolsSessionDomains;
+
+
+namespace Selenium4Sample
+{
+    public class Example
+    {
+
+        protected IDevToolsSession session;
+        protected IWebDriver driver;
+        protected DevToolsSessionDomains devToolsSession;
+
+        public void ConsoleLogTest()
+        {
+            driver = new ChromeDriver();
+            //Get DevTools
+            IDevTools devTools = driver as IDevTools;
+            //DevTools Session
+            session = devTools.GetDevToolsSession();
+
+            devToolsSession.Log.Enable();
+            devToolsSession.Log.EntryAdded += Log_EntryAdded;
+        }
+
```

Comment:
Extra spaces can be removed !

Reply:
> Hi @namilkimfree,
> Thanks for the PR.
> 
> I have a couple of suggestions on this PR:
> 
> 1. Code looks good to me, but it can be a shorter code snippet than the code in PR
> 2. Code needs to be changed in other translated pages too.
> 3. You need to also sign the CLA in order to get this PR merged

Hi @harsha509 

1. Code changed to short.
2. Changed  other translated pages too.